PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -172,6 +172,56 @@ def miniAOD_customizeCommon(process):
     del process.slimmedMETsNoHF.caloMET
     # ================== NoHF pfMET
 
+    #  ==================  CHSMET 
+    process.CHSCands = cms.EDFilter("CandPtrSelector",
+                                    src=cms.InputTag("packedPFCandidates"),
+                                    cut=cms.string("fromPV(0) > 0")
+                                    )
+    task.add(process.CHSCands)
+
+    process.pfMetCHS = cms.EDProducer("PFMETProducer",
+                                      src = cms.InputTag("CHSCands"),
+                                      alias = cms.string('pfMet'),
+                                      globalThreshold = cms.double(0.0),
+                                      calculateSignificance = cms.bool(False),
+                                      )
+    task.add(process.pfMetCHS)    
+
+    addMETCollection(process,
+                     labelName = "patChsMet",
+                     metSource = "pfMetCHS"
+                     )
+
+    process.patChsMet.computeMETSignificance = cms.bool(False)
```

Comment:
hi @zdemirag - please follow a convention here. Eg

CHS or Chs (seems CHS is the established one)
ChsMet or MetChs

(and similarly for TrkMet)

Reply:
The differences between patXXXMet and MetXXX is the pat vs reco modules. therefore the notation is self consistent.

Reply:
Its not XXXMet vs MetXXX - rather its "XXX" being consistent that I'm requesting.It looks like CHS is currently universal in CMSSW python:

[dlange@lxplus098 CMSSW_9_4_0_pre2]$ cmsglimpse CHS | grep python | wc
Your query may search about 72% of the total space! Continue? (y/n)y
   1251    5252  168118
[dlange@lxplus098 CMSSW_9_4_0_pre2]$ cmsglimpse Chs | grep python | wc
Your query may search about 72% of the total space! Continue? (y/n)y
      0       0       0